### PR TITLE
Add interpolation and velocity control for 3 grippers

### PR DESCRIPTION
**Pull request by nzlz**
Note that I had to add a `<id>` tag to urdf since we need to identify them in the plugin.

Adding static gripping capabilities is WIP on top of this in the `rework-with-interp-grip` branch.

### CODE REVIEW

**Comment by nzlz** [comment]
Also I have to mention that same as before, hande range goes from 0 to 0.025. Is this happening also in the real gripper? If its different, we should replicate it.

**Comment by YueErro** [comment]
I will take care of the <id> tag.
Unfortunately, on real driver is not the same i believe.
This measurements are fine, taken from the real robot.

**Comment by nzlz** [comment]
We should use same values as in real. We just need to know the max opening value for linear goal in hande.

File: robotiq_gripper_gazebo_plugins/include/robotiq_gripper_gazebo_plugins/RobotiqGripperPlugin.h [hunk 1]
```diff
@@ -64,6 +69,22 @@ namespace gazebo
...
+      const double MinVelocity_s140 = 30; //1.6;
+      const double MaxVelocity_s140 = 250; //0.19;
+      const double radius_s140 = 156; //mm
+      // S85 values. Min and max joint speed values:  20 to 150 mm/s
+      const double MinVelocity_s85 = 20; //1.5;
+      const double MaxVelocity_s85 = 150;// 0.2;
+      const double radius_s85 = 100; //mm
+      // S50 values. Min and max joint speed values:  20 to 150 mm/s
+      const double MinVelocity_s50 = 20;
+      const double MaxVelocity_s50 = 150;
```
Comment (YueErro) [inline robotiq_gripper_gazebo_plugins/include/robotiq_gripper_gazebo_plugins/RobotiqGripperPlugin.h]: Can we take these values from the urdf?

**Review by YueErro — CHANGES_REQUESTED** [review]

File: robotiq_gripper_gazebo_plugins/include/robotiq_gripper_gazebo_plugins/RobotiqGripperPlugin.h (hunk 1 above)
Comment (rkojcev) [reply robotiq_gripper_gazebo_plugins/include/robotiq_gripper_gazebo_plugins/RobotiqGripperPlugin.h]: that would be ideal instead of hardcoding them here.

**Comment by nzlz** [comment]
I guess add min, max, and radius to the plugin part of the urdf?

**Comment by rkojcev** [comment]
That is also an option @nzlz

**Comment by YueErro** [comment]
They are already there ([limit](https://github.com/AcutronicRobotics/robotiq_modular_gripper/blob/master/robotiq_hande_gripper_description/urdf/robotiq_hande.urdf.xacro#L98)).

**Comment by rkojcev** [comment]
Lets use these values then @nzlz and @YueErro

**Comment by nzlz** [comment]
We dont have lower velocity limits there. Also, which are the units? I guess revolute rad/s and prismatic m/s?

Also, can we trust these max values? Right now the s85 has a higher max limit in the urdf than the s140, that is incorrect. In the plugin I did the calculation using the radius of the angular movement, which is not a precise value. (measured in gazebo +-)

**Comment by YueErro** [comment]
The units and limits are [these](https://github.com/erlerobot/management/issues/1870#issuecomment-487074368) ones, taken from the real grippers, not from the real driver.

**Comment by YueErro** [comment]
Instead of taking them from the limit tag we can add each min-max tags in the plugin, same as [PID values](https://github.com/AcutronicRobotics/robotiq_modular_gripper/blob/master/robotiq_hande_gripper_description/urdf/robotiq_hande.urdf.xacro#L104-L106), and in the unit we want.

**Comment by nzlz** [comment]
Yes i think adding to the plugin part in urdf is the best option.

**Comment by nzlz** [comment]
Forgot to mention that I removed the `id` tag since we decided to use the following:

```xml
        <min_velocity>X</min_velocity>
        <max_velocity>X</max_velocity>
        <radius>X</radius> <!--Only for revolute joints-->
```

**Comment by ahcorde** [comment]
we need to resolve some issues to be able to merge with master @yue or @nzlz can you take care?

**Comment by nzlz** [comment]
Pushed merge. @YueErro can you review? 

Ill test everything again to make sure its working fine.

**Comment by nzlz** [comment]
Merge master **not** working. Don't merge. Fix WIP

**Comment by YueErro** [comment]
The plugin is duplicated. It should be removed from the grippers.urdf.xacro and regenerate the .urdf files.

**Comment by nzlz** [comment]
## PR tests

- [x] S140
- [x] S85
- [x] S50 (hande)

---
Example for S140 (revolute, so using angular position, linear is not used)
terminal1
```bash
ros2 launch robotiq_gazebo robotiq_140.launch.py
```
Terminal2

```bash
ros2 service call /hrim_actuator_gripper_000000000004/fingercontrol hrim_actuator_gripper_srvs/ControlFinger "{goal_linearposition: 0.0, goal_angularposition: 0.0, goal_velocity: 100.0}"
```

**Review by YueErro — APPROVED** [review]

**Review by ahcorde — APPROVED** [review]

**Pull request merged**